xml/art_sles_amd-sev.xml

```diff
@@ -0,0 +1,431 @@
+   Encryption of computer data stored on disk is a widely-deployed feature.
+   However, data in RAM is stored in the clear. This can leave that data
+   vulnerable to software or hardware probing by intruders into the host system.
+   New persistent-memory technology exacerbates this problem, since an NVDIMM
+   (non-volatile memory module) could be physically removed from a system and
+   the data on it will remain intact, similarly to that on a hard drive or SSD.
+   Without encryption, any stored information - such as sensitive data,
+   passwords, or secret keys - could easily be compromised.
+  </para>
+  <para>   
+   AMD's SEV (Secure Encrypted Virtualization) is a technology to protect Linux
+   KVM virtual machines by transparently encrypting the memory of each VM with a
+   unique key. SEV can also calculate a signature of the memory contents, which
+   can be sent to the VM's owner as an attestation that the memory was encrypted
+   correctly by the firmware. SEV is particularly applicable to cloud computing
+   since it can reduce the amount of trust VMs need to place in the hypervisor
+   and administrator of their host system.
+  </para>
+  <para>
+   In &sle; 12 SP4 and above, and a forthcoming maintenance release of &sle; 15,
+   the kernel, QEMU, and <literal>libvirt</literal> support creating and
+   managing VMs whose memory is encrypted using AMD's SEV technology. Currently
+   the technology is only available as technical preview, but it will be fully
+   supported in future versions of &sle;.
+  </para>
+ </sect1>
+ 
+ <sect1 xml:id="sec.amd-sev.hostreqs">
+  <title>VM Host Requirements</title>
+  <para>
+   The VM host hardware must support AMD's SEV technology. Support for SEV can
+   be detected by checking that the <literal>sev</literal> attribute is present
+   in the capabilities of <literal>libvirt</literal> and that its value is set
+   appropriately:
+  </para>
+<screen><![CDATA[
+<domainCapabilities>
+ ...
+ <features>
+  ...
+  <sev supported='yes'/>
+  ...
+  </sev>
+ </features>
+</domainCapabilities>
+]]></screen>
+  <para>
+   Additionally, ensure that the <literal>kvm_amd</literal> kernel module has
+   the <literal>sev</literal> parameter enabled:
+  </para>
+  <screen>/sys/module/kvm_amd/parameters/sev = 1</screen>
+ </sect1>
+ 
+ <sect1 xml:id="sec.amd-sev.vmreqs">
+  <title>Virtual Machine Requirements</title>
+  <para>
+   The VM must be the modern <literal>Q35</literal> machine type and must use
+   UEFI firmware.
+  </para>
+  <note>
+   <para>
+    The Q35 machine type does not have an IDE controller and does not support IDE disks.
+   </para>
+  </note>
+  <para>
+   Currently <literal>virtio-blk</literal> disks are not supported. <literal>virtio-scsi</literal>
+   and SATA disks are supported and work as expected. All <literal>virtio</literal>
+   devices need to be configured with the <literal>iommu='on'</literal>
+   attribute in their <literal>&lt;driver&gt;</literal> configuration. In
```

Comment:
As this is XML, I'd suggest to use `<tag class="starttag">driver</tag>` as this will print the pointy brackets automatically for you. :smiley: 

You can use them whenever there is XML needed. Pro tip: the `class` attribute allows more values (empty tag, end tags, PIs, entities, XML comments etc.). :wink: 

There are other occurances in this file, so I will not mark each line individually.

Reply:
I _think_ I have changed all of these now.

Reply:
One thing @tomschr  -- I do not see the pointy brackets. Did I do something wrong?